helix-core/src/main/java/org/apache/helix/manager/zk/ZkBucketDataAccessor.java

```diff
@@ -73,8 +77,22 @@
    * @param versionTTL in ms
    */
   public ZkBucketDataAccessor(String zkAddr, int bucketSize, long versionTTL) {
-    _zkClient = DedicatedZkClientFactory.getInstance()
-        .buildZkClient(new HelixZkClient.ZkConnectionConfig(zkAddr));
+    if (Boolean.getBoolean(SystemPropertyKeys.MULTI_ZK_ENABLED)) {
```

Comment:
nit, just to confirm, I think we are migrating all the SystemPropertyKeys to someone station HelixProperty solution. Will this be also migrated to it once it's done?

Reply:
This config should follow wherever all Helix system config properties go. I am not sure about what you mean by HelixProperty solution, but if there's an agreed-upon place we should group all configs in, then this should go with them.